### DISCUSSION TITLE
Support for Std.dev in the legends

### DISCUSSION THREAD

**Discussion by selvavm** [grafana/grafana#46313] [Dashboards, Panels & Visualizations]
Can we include the std.dev in the legend? I think it has to be implemented as a transform function but don't know how to do

**Comment by ryantxu** [comment]
Added in 9.1. https://github.com/grafana/grafana/pull/52769

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
